.github/workflows/nodejs-ci.yml

```diff
@@ -12,7 +12,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [14.x, 16.x, 18.x]
+        node-version: [14.x, 16.x, 18.x, 20.x]
```

Comment:
I forget what make us drop test version. 14 eol April. Maybe we hanging it there until github drop it from matrix?